runtime/dgsh_tee.sh

```diff
@@ -8,7 +8,7 @@ PASH_TOP=${PASH_TOP:-$(git rev-parse --show-toplevel)}
 
 cleanup()
 {
-    kill -SIGPIPE $dgsh_tee_pid > /dev/null 2>&1
+    kill -SIGKILL $dgsh_tee_pid > /dev/null 2>&1
```

Comment:
Maybe we want `SIGTERM`, which is more standard termination.

Reply:
Minor comment: I agree too with going with `SIGTERM`, because it gives some time for the process itself to exit gracefully, whatever that means for each process. As commands be have `SIGTERM` handlers though (possibly ignoring it/blocking it),  we have to be prepared to do another final round of cleanup with `SIGKILL`.

Reply:
I agree with the comment about handlers but note that this `kill` here is only performed on `dgsh-tee` and therefore is not meant to be generally applicable/generalizable.